rest-client/src/main/java/org/sagebionetworks/bridge/rest/UserSessionInterceptor.java

```diff
@@ -135,7 +181,21 @@ private boolean isSuccessfulSignOut(Request request, Response response) {
         return (SIGN_OUT_PATH_SEGMENT.equals(pathSeg) && response.code() == 200);
     }
 
-    private UserSessionInfo getUserSessionInfo(String bodyString) throws IOException {
-        return RestUtils.GSON.fromJson(bodyString, UserSessionInfo.class);
+    private boolean returnsMySessionInfo(Request request) {
+        if (PARTICIPANT_SELF_PATH.equals(request.url().encodedPath())
+                && HTTP_METHOD_POST.equals(request.method())) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
I did, set up a new machine. Dwayne sent me his a while back, let me sets that up